src/Files/TemporaryFileFactory.php

```diff
@@ -46,7 +46,7 @@ public function make(string $fileExtension = null): TemporaryFile
      */
     public function makeLocal(string $fileName = null, string $fileExtension = null): LocalTemporaryFile
     {
-        if (!file_exists($this->temporaryPath) && !mkdir($concurrentDirectory = $this->temporaryPath, config('excel.temporary_files.local_permissions.dir', 0777)) && !is_dir($concurrentDirectory)) {
+        if (!file_exists($this->temporaryPath) && !mkdir($concurrentDirectory = $this->temporaryPath, config('excel.temporary_files.local_permissions.dir', 0777, true)) && !is_dir($concurrentDirectory)) {
```

Comment:
@iftaras Is the position of third flag correct?

Reply:
@majumawm no it's not. Thanks for pointing that out.